### ISSUE TITLE
Made Quick Capture under a heading in a note 

### ISSUE THREAD

**Issue by Feranmi-Star90** [tmfelwu/obsidian-inbox#1]
Hi, 

Thanks to the developer who made this plugin, but can I add content under a heading in the note I select?

**Comment by tmfelwu** [comment]
The current functionality allows you to do that. You provide the title and the content. The content is added below the heading. You can even use markdown in the `Enter note content box.`

Are you looking for something else?

**Comment by Feranmi-Star90** [comment]
Yes, I did try that but nothing shows in the note I choose from the dropdown menu. But, it does work when I create the new note and add the content underneath. I don't know what I am doing wrong.

**Comment by tmfelwu** [comment]
Can you please share a video if possible.

**Comment by Feranmi-Star90** [comment]
I apologise for been so late in replying, but this is the issue I have been having:


https://github.com/tmfelwu/obsidian-inbox/assets/79968557/b756a990-0630-4df1-9794-4431461129d1

**Comment by tmfelwu** [comment]
Right this functionality is not implemented yet. If a not exists it just
helps navigating to that note. What could be other alternative?

1. Add at the end of the note?
2. Add at the beginning of the note?

And provide a setting to seitch between the two.

Can you think of something else?
